tests/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
```

Comment:
Let's move the docker-compose file to folder `tests/mysql` and `test/postgres` with different DB starting.

Reply:
and change the folder name `tests` to `integration-tests`